packages/fxa-content-server/app/scripts/templates/sign_in_password.mustache

```diff
@@ -41,26 +41,26 @@
          - showing the doorhanger. -->
       <input class="hidden" required />
 
-      <div class="button-row">
-        <button id="submit-btn" {{^isPasswordNeeded}}class="use-logged-in"{{/isPasswordNeeded}} type="submit">{{#t}}Sign in{{/t}}</button>
+      <div class="flex">
+        <button id="{{^isPasswordNeeded}}use-logged-in"{{/isPasswordNeeded}}{{#isPasswordNeeded}}submit-btn{{/isPasswordNeeded}}" class="cta-primary cta-xl {{^isPasswordNeeded}}use-logged-in{{/isPasswordNeeded}}" type="submit">{{#t}}Sign in{{/t}}</button>
       </div>
 
-        <div class="tos-pp faint">
+        <div id="tos-pp" class="text-grey-500 my-5 text-xs">
           {{#isPocketClient}}
             {{#unsafeTranslate}}
               By proceeding, you agree to:<br />
-              Pocket’s <a id="pocket-tos" href="https://getpocket.com/en/tos/">Terms of Service</a> and <a id="pocket-pp" href="https://getpocket.com/en/privacy/">Privacy Notice</a><br />
-              Firefox’s <a id="fxa-tos" href="/legal/terms">Terms of Service</a> and <a id="fxa-pp" href="/legal/privacy">Privacy Notice</a>
+              Pocket’s <a class="link-grey" id="pocket-tos" href="https://getpocket.com/en/tos/">Terms of Service</a> and <a id="pocket-pp" class="link-grey" href="https://getpocket.com/en/privacy/">Privacy Notice</a><br />
+              Firefox’s <a class="link-grey" id="fxa-tos" href="/legal/terms">Terms of Service</a> and <a class="link-grey" id="fxa-pp" href="/legal/privacy">Privacy Notice</a>
```

Comment:
Probably nothing, but just noticed the string on L57 has a trailing period, but L53 doesn't.